src/test/java/org/jabref/logic/bibtex/BibEntryWriterTest.java

```diff
@@ -515,24 +515,6 @@ void doNotWriteEmptyFields() throws IOException {
         assertEquals(expected, actual);
     }
 
-    @Test
-    void trimFieldContents() throws IOException {
```

Comment:
Can we keep this test --> maybe for the whitespace removal thing?

Reply:
I moved the test to the database writer as this is the new place where the field content is trimmed.
https://github.com/JabRef/jabref/pull/5838/files#diff-128297a947efc0b9e025fd5541c72376R650